pkg/arvo/app/group-view.hoon

```diff
@@ -152,43 +152,60 @@
           %+  poke-our:(jn-pass-io /pull-groups)  %group-pull-hook 
           pull-hook-action+!>([%add ship rid])
       (tx-progress %added)
-      ::
+    ::
         %pull-groups
       ?>  ?=(%poke-ack -.sign)
       (ack +.sign)
-      ::
+    ::
         %groups
       ?+  -.sign  !!
         %fact  (groups-fact +.sign)
         %watch-ack  (ack +.sign)
         %kick  watch-groups
       ==
-      ::
+    ::
         %pull-md
       ?>  ?=(%poke-ack -.sign)
       (ack +.sign)
-      ::
+    ::
+        %pull-co
+      ?>  ?=(%poke-ack -.sign)
+      (ack +.sign)
```

Comment:
I don't think these contact pokes should use the +ack gate, because it'll report the join transaction as failed if the poke nacks, which isn't necessarily true. Maybe just +ignore-ack where
```hoon
++  ignore-ack
  |=  err=(unit tang)
  ?~  err  jn-core
  %-  (slog u.err)
  jn-core
```

Reply:
I think the transaction should fail tho. Talking about this on call with you rn, but documenting discussion lol